projects/1.16/assets/simply-cats/simplycats/lang/zh_cn.json

```
@@ -0,0 +1,248 @@
+    "block.simplycats.brown_cat_tree_bed": "棕色猫窝",
+    "block.simplycats.green_cat_tree_bed": "绿色猫窝",
+    "block.simplycats.red_cat_tree_bed": "红色猫窝",
+    "block.simplycats.black_cat_tree_bed": "黑色猫窝",
+
+    "block.simplycats.white_cat_tree_box": "白色树屋",
+    "block.simplycats.orange_cat_tree_box": "橙色树屋",
+    "block.simplycats.magenta_cat_tree_box": "品红色树屋",
+    "block.simplycats.light_blue_cat_tree_box": "淡蓝色树屋",
+    "block.simplycats.yellow_cat_tree_box": "黄色树屋",
+    "block.simplycats.lime_cat_tree_box": "黄绿色树屋",
+    "block.simplycats.pink_cat_tree_box": "粉色树屋",
+    "block.simplycats.gray_cat_tree_box": "灰色树屋",
+    "block.simplycats.light_gray_cat_tree_box": "淡灰色树屋",
+    "block.simplycats.cyan_cat_tree_box": "青色树屋",
+    "block.simplycats.purple_cat_tree_box": "紫色树屋",
+    "block.simplycats.blue_cat_tree_box": "蓝色树屋",
+    "block.simplycats.brown_cat_tree_box": "棕色树屋",
+    "block.simplycats.green_cat_tree_box": "绿色树屋",
+    "block.simplycats.red_cat_tree_box": "红色树屋",
+    "block.simplycats.black_cat_tree_box": "黑色树屋",
+
+    "block.simplycats.white_cat_tree_post": "白色猫爬架",
+    "block.simplycats.orange_cat_tree_post": "橙色猫爬架",
+    "block.simplycats.magenta_cat_tree_post": "品红色猫爬架",
+    "block.simplycats.light_blue_cat_tree_post": "淡蓝色猫爬架",
+    "block.simplycats.yellow_cat_tree_post": "黄色猫爬架",
+    "block.simplycats.lime_cat_tree_post": "黄绿色猫爬架",
+    "block.simplycats.pink_cat_tree_post": "粉色猫爬架",
+    "block.simplycats.gray_cat_tree_post": "灰色猫爬架",
+    "block.simplycats.light_gray_cat_tree_post": "淡灰色猫爬架",
+    "block.simplycats.cyan_cat_tree_post": "青色猫爬架",
+    "block.simplycats.purple_cat_tree_post": "紫色猫爬架",
+    "block.simplycats.blue_cat_tree_post": "蓝色猫爬架",
+    "block.simplycats.brown_cat_tree_post": "棕色猫爬架",
+    "block.simplycats.green_cat_tree_post": "绿色猫爬架",
+    "block.simplycats.red_cat_tree_post": "红色猫爬架",
+    "block.simplycats.black_cat_tree_post": "黑色猫爬架",
+
+    "__comment": "Creative Tab Name",
+    "itemGroup.simplycats.tab": "Simply Cats",
+
+    "__comment": "Egg and Entity Names",
+    "entity.simplycats.cat": "猫",
+    "entity.simplycats.cat.unknown_owner": "未知饲主",
+    "entity.simplycats.cat.untamed": "未驯服",
+    "entity.minecraft.villager.simplycats.shelter_worker": "动物保护员",
+
+    "__comment": "Book Data",
+    "book.index_page.info": "你好! 这是Simply Cats猫咪图鉴的索引页,试着对猫咪潜行+右键使用吧！",
+    "book.genetics.eye_color": "虹膜颜色: %s",
+    "book.genetics.fur_length": "毛发长度: %s",
+    "book.genetics.eumelanin": "真黑素: %s",
+    "book.genetics.phaeomelanin": "褐黑素: %s",
+    "book.genetics.dilute": "稀释基因: %s",
+    "book.genetics.dilute_modifier": "稀释修饰基因: %s",
+    "book.genetics.agouti": "虎斑毛发基因: %s",
+    "book.genetics.tabby": "斑纹: %s",
+    "book.genetics.spotted": "点斑: %s",
+    "book.genetics.ticked": "细纹斑: %s",
+    "book.genetics.inhibitor": "Inhibitor: %s",
+    "book.genetics.colorpoint": "重点色: %s",
+    "book.genetics.white": "显性白基因: %s",
+    "book.genetics.bobtail": "短尾: %s",

```

```suggestion
    "book.genetics.eye_color": "虹膜颜色：%s",
    "book.genetics.fur_length": "毛发长度：%s",
    "book.genetics.eumelanin": "真黑素：%s",
    "book.genetics.phaeomelanin": "褐黑素：%s",
    "book.genetics.dilute": "稀释基因：%s",
    "book.genetics.dilute_modifier": "稀释修饰基因：%s",
    "book.genetics.agouti": "虎斑：%s",
    "book.genetics.tabby": "斑纹：%s",
    "book.genetics.spotted": "点斑：%s",
    "book.genetics.ticked": "细纹斑：%s",
    "book.genetics.inhibitor": "抑制基因：%s",
    "book.genetics.colorpoint": "重点色：%s",
    "book.genetics.white": "显性白基因：%s",
    "book.genetics.bobtail": "短尾：%s",
```
从Phaeomelanin后面开始只做了简单的查找
抓个在这部分了解更全面的来审核（）

---

嘿嘿，辛苦了
冒号已修改